vocabularies/Org.OData.Core.V1.json

```diff
@@ -235,6 +235,97 @@
                 }
             ]
         },
+        "ValueException": {
+            "$Kind": "Term",
+            "$Nullable": true,
+            "$Type": "Core.ValueExceptionType",
+            "@Core.Description": "The annotated numeric value is problematic"
+        },
+        "ExceptionType": {
+            "$Kind": "ComplexType",
+            "$Abstract": true,
+            "info": {
+                "$Nullable": true,
+                "$Type": "Core.Message",
+                "@Core.Description": "Information on the (cause of) the exception"
+            }
+        },
+        "ValueExceptionType": {
+            "$Kind": "ComplexType",
+            "$BaseType": "Core.ExceptionType",
+            "value": {
+                "@Core.Description": "The actual numeric value"

```

Comment:
The value property is optional right, so isn't there a "$Nullable": true missing here?